internal/app/referenceclient/impl.go

```diff
@@ -360,7 +361,7 @@ func (i *invoker) clientStream(
 		// so that we can get the headers from the Meta property. Then,
```

Comment:
The comment should also be `trailers` right (like the change above)?

Reply:
Yes, good catch!

Reply:
```suggestion
		// so that we can get the trailers from the Meta property. Then,
```